plugins/module_utils/data_set.py

```diff
@@ -1035,7 +1037,8 @@ def create(
                     return 0
             raise DatasetCreateError(
                 name,
-                msg="Unable to verify the data set was created. Received DatasetVerificationError from ZOAU.",
+                msg="Unable to verify the data set was created. Received DatasetVerificationError from ZOAU." +
+                str(formatted_args) + "--rf originally=" + record_format + ".",
```

Comment:
Is this a new error message or just part of the debugging print process ? If this is going to be customer-facing I think we could rephrase it.

Reply:
I added the originally section for debugging.  I have removed it.  Good catch!